Cargo.toml

```diff
@@ -19,7 +19,7 @@ targets = [
 ]
```

Comment:
Do we need to add `i686-unknown-freebsd`?

Reply:
That's just for docs.  Do you think we should build docs for i686-unknown-freebsd too?

Reply:
ah right! my bad.